pkgs/by-name/hu/hunspell/wrapper.nix

```diff
@@ -0,0 +1,18 @@
+{
+  stdenv,
+  lib,
+  hunspell,
+  makeBinaryWrapper,
+  dicts ? [ ],
+}:
+let
+  searchPath = lib.makeSearchPath "share/hunspell" dicts;
+in
+stdenv.mkDerivation {
+  inherit (lib.appendToName "with-dicts" hunspell) name;
+  nativeBuildInputs = [ makeBinaryWrapper ];
+  buildCommand = ''
+    wrapProgram $out/bin/hunspell --prefix DICPATH : ${lib.escapeShellArg searchPath}
```

Comment:
i think this change is wrong. e.g. this command is buildable on master, but not on this PR branch:
```
echo ':l\n :b hunspellWithDicts [ hunspellDicts.en_US ]' | nix repl
```

Reply:
Should be fixed

Reply:
I also added a test for the wrapper in the main package to catch anything like this in the future